rollup.config.js

```diff
@@ -12,37 +15,66 @@ const babelConfig = require('./babel.config')
 
 const env = process.env.NODE_ENV
 const version = process.env.npm_package_version
-const [, format] = process.env.npm_lifecycle_event.split(':')
+const [, packageName] = process.env.npm_package_name.split('@commercetools/')
 const extensions = ['.js', '.jsx', '.ts', '.tsx']
+const plugins = [
+  json(),
+  replace({
+    'process.env.NODE_ENV': JSON.stringify(env),
+    VERSION: `'${version}'`,
+  }),
+  resolve({
+    extensions,
+    mainFields: ['module', 'main', 'jsnext'],
+    preferBuiltins: true,
+    modulesOnly: true,
+  }),
+  commonjs({
+    include: ['node_modules/**'],
+  }),
+  babel({
+    exclude: ['node_modules/**'],
+    runtimeHelpers: true,
+    extensions,
+    ...babelConfig,
+  }),
+  filesize(),
+].filter(Boolean)
 
-const config = {
-  output: {
-    sourcemap: true,
+const createConfig = cliArgs => [
+  {
+    input: cliArgs.input,
+    output: {
+      format: 'umd',
+      file: `dist/${packageName}.umd.js`,
```

Comment:
Are we only building one UMD bundle now?

Reply:
Look below 😊
The second one is minified

Reply:
🤦‍♂